SPECS/edk2/edk2.spec

```diff
@@ -27,39 +27,47 @@ Distribution:   Azure Linux
 # https://github.com/netwide-assembler/nasm/pull/3
 ExclusiveArch: x86_64
```

Comment:
Any chance to get this to work for aarch64 as well? In 2.0 edk2 builds only for x64 because the arm builds fail. But with 3.0 upgraded packages maybe there is a chance to get it working for aarch64 as well.

Reply:
tried it, build on arm still failed for missing dep `nasm`